client/tests/integration/smartcontracts/validator_with_custom_token/src/lib.rs

```diff
@@ -152,8 +161,16 @@ impl CustomValidator {
     }
 }
 
-impl Visit for CustomValidator {
-    fn visit_register_domain(&mut self, authority: &AccountId, _register_domain: Register<Domain>) {
+macro_rules! defaults {
```

Comment:
This macro seems to be use more than once. Maybe it can later be extracted into a top level module?

Reply:
there is #3856